backend/routes/seatbid.js

```diff
@@ -179,20 +202,34 @@ router.post('/accept', authenticateUser, async (req, res) => {
  * @desc Get all active biddings for a specific flight
  * @access Private
  */
-router.get('/active/:flightId', authenticateUser, (req, res) => {
+router.get('/active/:flightId', authenticateUser, async (req, res) => {
     const { flightId } = req.params;
-    const flightBiddings = Array.from(activeBiddings.values())
-        .filter(bidding => bidding.flightId === flightId && bidding.status === 'ACTIVE')
-        .map(bidding => ({
-            biddingId: `${bidding.flightId}-${bidding.seatNumber}`,
+
+    try {
+        const activeBiddings = await prisma.bidding.findMany({
```

Comment:
It would be good if we can check here whether bidding was expired (in case the server was killed and we couldn't expire that). Also, we could create a cron job to auto-delete "corrupted" biddings and clean those up.

Not necessary to make those changes for your TC, I'm just explaining how this would be handled in a final product.